### PR TITLE
gnome: remove deprecated/obsolete technology

### PR DESCRIPTION
**Pull request by maxice8**
See body of each commit for what to look for.

### CODE REVIEW

File: main/gconf/APKBUILD [hunk 1]
```diff
@@ -22,7 +20,10 @@ build() {
 		--build=$CBUILD \
 		--host=$CHOST \
 		--prefix=/usr \
-		--sysconfdir=/etc
+		--sysconfdir=/etc \
+		--disable-static \
```
Comment (tcely) [inline main/gconf/APKBUILD]: Splitting out `-static` would be better.

Comment (maxice8) [reply main/gconf/APKBUILD]: I don't think there is much value in splitting out static libraries of gnome stuff

**Comment by ncopa** [comment]
Seems like py-libgnome still depends on libgnome?

**Comment by ncopa** [comment]
what about `community/libgnome-keyring` which has `makedepends=libgnome-dev`?

**Comment by maxice8** [comment]
> what about `community/libgnome-keyring` which has `makedepends=libgnome-dev`?

https://github.com/alpinelinux/aports/pull/7416

**Comment by maxice8** [comment]
> Seems like py-libgnome still depends on libgnome?

https://github.com/alpinelinux/aports/pull/7418

**Pull request closed without merging**